src/registrar/utility/errors.py

```diff
@@ -77,17 +77,15 @@ class FSMErrorCodes(IntEnum):
         - 1 APPROVE_DOMAIN_IN_USE The domain is already in use
         - 2 NO_INVESTIGATOR No investigator is assigned
         - 3 INVESTIGATOR_NOT_STAFF Investigator is a non-staff user
-        - 4 INVESTIGATOR_NOT_SUBMITTER The form submitter is not the investigator
```

Comment:
removed references to INVESTIGATOR_NOT_SUBMITTER since it is not used anywhere and has no reason to exist anymore after deprecating submitter